cocos/ui/UIListView.h

```diff
@@ -244,6 +244,18 @@ class CC_GUI_DLL ListView : public ScrollView
      */
     float getItemsMargin()const;
     
+    /**
+     * Set time to scroll between items (see scrollToItem)
+     * @param time
+     */
+    void  setScrollDuration(const float time);
```

Comment:
I mark `const` read only parameters for two reasons:
- Sometime compilers can optimize `const` parameters
- Avoid programmer mistakes (as unwanted parameter modification).
Anyway I see that cocos2d-x programmers typically does not mark `const` to read only parameter. Then I will delete the `const` for consistency